plugin/vim-gh-line.vim

```diff
@@ -77,6 +77,10 @@ func! s:gh_line() range
           let lineRange = 'L' . a:firstline . "-L" . a:lastline
       endif
       let url = origin . blob . commit . relative . '#' . fileName . lineRange
+     elseif exists("g:gh_gitlab_domain")
```

Comment:
Would it be better to check `g:gh_gitlab_domain` against `origin`?

Reply:
Origin maybe self setted, like deploy a gitlab server and set hostname is 'code-base.com'

Reply:
Maybe check `exists("g:gh_gitlab_domain")` and `match(origin, "g:gh_gitlab_domain")` would be better?

Reply:
@laixintao that's what I mean 👍